doc/source/rllib-concepts.rst

```diff
@@ -622,4 +622,4 @@ Example of three equivalent ways of interacting with the PPO trainer, all of whi
 .. code-block:: python
 
     from ray import tune
-    tune.run(PPOTrainer, config={"env": "CartPole-v0", "train_batch_size": 4000})
```

Comment:
this should work without this change

Reply:
Oh, right. Sorry.
I'll close this PR as the other changes seem too negligible. Maybe I'll open a PR later with more doc fixes.